generators/app/index.js

```diff
@@ -214,7 +240,10 @@ module.exports = yeoman.Base.extend({
 
     this.fs.copy(
       this.templatePath('LICENSE'),
-      this.destinationPath(this.props.baseDirectory + 'LICENSE')
+      this.destinationPath(this.props.baseDirectory + 'LICENSE'),
+      {
+        currentYear: this.props.currentYear
```

Comment:
![](https://media.giphy.com/media/xKy2w6LehxxHa/giphy.gif)

Reply:
Are we ok with this? I left the comment on another code review about this, and @bjankord said he had to consult with James on what year the copyright is valid for.

Reply:
Looks like there is already an issue logged for it. https://github.com/cerner/terra-framework/issues/65